tests/phpunit/tests/query/stickies.php

```diff
@@ -126,4 +126,27 @@ public function test_stickies_should_obey_parameters_from_the_main_query() {
 		$this->assertTrue( $sticky_query_vars['ignore_sticky_posts'] );
 		$this->assertTrue( $sticky_query_vars['no_found_rows'] );
 	}
+
+	/**
+	 * @ticket 36907
+	 */
+	public function test_stickies_should_limit_query() {
+		$sticky_count = 5;
```

Comment:
```suggestion
		$sticky_count = 6;
```

Are you able to make this six or more, basically anything above five so we can be sure it's above the `get_posts()` default limit.